src/sentry_plugins/github/testutils.py

```diff
@@ -16,11 +16,11 @@
       "id": "133d60480286590a610a0eb7352ff6e02b9674c4",
       "tree_id": "f9d2a07e9488b91af2641b26b9407fe22a451433",
       "distinct": true,
-      "message": "Update README.md (àgain)",
+      "message": "Update README.md (again)",
```

Comment:
`b"""` isn't a raw string, it's a bytestring. The comment above says to keep as a raw string... _shrugs_. Could you change that to `r"""` and undo the changes to `à`? I bet the linter won't complain.